console/templates/newaccount-notification-template.txt

```diff
@@ -1,11 +1,16 @@
 Dear admin,
 
-A new user signed up on {publicUrl}/ !
+A new {oauth2} user signed up on {publicUrl}/ !
 
 User name: {name}
 User email: {email}
-User ID: {uid}
-User Organization: {org}
+{uid_msg}{org_msg}{provider_msg}
+
+You can manager {name} account parameters on {publicUrl}/console/manager/users/{user_id}/infos !
```

Comment:
`Manage account on {publicUrl}/console/manager/users/{user_id}/infos` will be more neutral.

Reply:
code updated

Reply:
No trailing "!" please